lib/compat/wordpress-5.9/block-editor-settings.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Adds settings to the block editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds styles and __experimentalFeatures to the block editor settings.
+ *
+ * @param array $settings Existing block editor settings.
+ *
+ * @return array New block editor settings.
+ */
+function gutenberg_get_block_editor_settings( $settings ) {
+	// Set what is the context for this data request.
+	$context = 'other';
```

Comment:
In [WordPress core](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/block-editor.php#L300), we don't check who is the caller. Both `styles` and `settings` are always enqueued for any caller.